### ISSUE TITLE
[Bug] First run broken

### ISSUE THREAD

**Issue by miguelangel-nubla** [brenner-tobias/addon-cloudflared#63]
**Describe the bug**
On clean install and first run tunnel.json does not have TunnelName property.
Maybe the new version of cloudflared creates the file diferently?

After copy pasting tunnel.json from the logs, adding "TunnelName":"homeassistant" and restarting it works fine.

**To Reproduce**
Clean install, config and enable debug logging, first run.
Authorized correctly, then error:
```
DEBUG: Tunnnel name read from file: null
WARNING: Tunnel name in file does not match config, removing tunnel file
and exit.
```

**Logs**
```
[s6-init] making user provided files available at /var/run/s6/etc...exited 0.
[s6-init] ensuring user provided files have correct perms...exited 0.
[fix-attrs.d] applying ownership & permissions fixes...
[fix-attrs.d] done.
[cont-init.d] executing container initialization scripts...
[cont-init.d] 00-banner.sh: executing... 
-----------------------------------------------------------
 Add-on: Cloudflared
 Use a Cloudflared tunnel (formerly Argo Tunnel) to remotely connect to Home Assistant without opening any ports
-----------------------------------------------------------
 Add-on version: 0.5.2
 You are running the latest version of this add-on.
 System: Home Assistant OS 7.4  (amd64 / qemux86-64)
 Home Assistant Core: 2022.3.1
 Home Assistant Supervisor: 2022.01.1
-----------------------------------------------------------
 Please, share the above information when looking for help
 or support in, e.g., GitHub, forums or the Discord chat.
-----------------------------------------------------------
[cont-init.d] 00-banner.sh: exited 0.
[cont-init.d] 00-cloudflared-log.sh: executing... 
[cont-init.d] 00-cloudflared-log.sh: exited 0.
[cont-init.d] 01-log-level.sh: executing... 
Log level is set to DEBUG
[cont-init.d] 01-log-level.sh: exited 0.
[cont-init.d] 10-cloudflared-config.sh: executing... 
[18:01:02] INFO: Checking Add-on config...
[18:01:03] INFO: Checking for existing certificate...
[18:01:03] NOTICE: No certificate found
[18:01:03] INFO: Creating new certificate...
[18:01:03] NOTICE: 
[18:01:03] NOTICE: Please follow the Cloudflare Auth-Steps:
[18:01:03] NOTICE: 
Please open the following URL and log in with your Cloudflare account:
https://dash.cloudflare.com/argotunnel?callback=https%3A%2F%2Flogin.cloudflareaccess.org%2F63dYTIhGuV#######redacted#######
Leave cloudflared running to download the cert automatically.
You have successfully logged in.
If you wish to copy your credentials to a server, they have been saved to:
/root/.cloudflared/cert.pem
[18:01:45] INFO: Authentication successfull, moving auth file to the '/data' folder
[18:01:45] INFO: Checking for existing certificate...
[18:01:45] INFO: Existing certificate found
[18:01:45] INFO: Checking for existing tunnel...
[18:01:45] NOTICE: No tunnel file found
[18:01:45] INFO: Creating new tunnel...
Tunnel credentials written to /data/tunnel.json. Keep this file secret. To revoke these credentials, delete the tunnel.
Created tunnel homeassistant with id #######redacted#######-b94a-b74e0eb05301
Tunnel Token: eyJhIjoiOGQyOTQwYTUzODE#######redacted#######
[18:01:46] DEBUG: Created new tunnel: {"AccountTag":"#######redacted#######8e5e1815","TunnelSecret":"#######redacted#######1DTiKc6Eacqw63A=","TunnelID":"#######redacted#######-b94a-b74e0eb05301"}
[18:01:46] INFO: Checking for existing tunnel...
[18:01:47] INFO: Existing tunnel with ID #######redacted#######-b94a-b74e0eb05301 found
[18:01:47] INFO: Checking if existing tunnel matches name given in config
[18:01:47] DEBUG: Tunnnel name read from file: null
[18:01:47] WARNING: Tunnel name in file does not match config, removing tunnel file
[18:01:47] FATAL: Failed to create tunnel

```

**Comment by elcajon** [comment]
Thanks @miguelangel-nubla for pointing this out. Seems to be an issue related to the latest cloudflared version. Should be fixed with #64

**Comment by darrylgibbs** [comment]
@elcajon any idea when you guys will be able to push the fix?

**Comment by felipecrs** [comment]
@darrylgibbs they are actively working on it. And usually @elcajon and @brenner-tobias are quite fast.

**Comment by darrylgibbs** [comment]
@felipecrs apologies if that came across rude, I'm just really keen to try this thing..ha ha.  I discovered it after hours of searching for the solution when I had the same issue.

Thanks for the amazing work all! @felipecrs @elcajon

**Comment by elcajon** [comment]
PR is ready, just needs to be re-approved. 😉

**Comment by brenner-tobias** [comment]
Should be fixed in v0.5.3, please re-open if this does not help. Thanks a lot @elcajon
